apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+// import { Notification } from
+import { View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+// import { ColorValue } from 'react-native';
```

Comment:
Nit: remove commented out lines. 

We can work on importing `Notification` from it's own package as a followup PR =)